createwindow/createwindow_test.go

```diff
@@ -0,0 +1,29 @@
+package createwindow
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestDisableCrashedBubble(t *testing.T) {
+	content := []byte("\"exit_type\":\"Crashed\"")
```

Comment:
Use a realistic data file. Copy from your chrome session preferences file, and create a test data file.